labeling/codebase/localcodebase_test.go

```diff
@@ -85,11 +91,6 @@ func TestLocalCodebase_FindFile(t *testing.T) {
 			glob:         "localcodebase_test.go",
 			expectedPath: "localcodebase_test.go",
 			expectErr:    false,
-		}, {
-			name:      "find.me not found in current dir",
-			BasePath:  ".",
-			glob:      "find.me",
-			expectErr: true,
 		},
```

Comment:
Can we add a test with multiple globs being passed in?

Reply:
Yes, good point. Done